.github/workflows/ci.yml

```diff
@@ -464,10 +461,10 @@ jobs:
         run: |
           set -euxo pipefail
           cargo nextest run -p tokio --all-features --target ${{ matrix.target }}
-          cargo test --doc -p tokio --all-features --target ${{ matrix.target }} -- --test-threads 1
+          cargo test --doc -p tokio --all-features --target ${{ matrix.target }}
         env:
           RUST_TEST_THREADS: 1
-          RUSTFLAGS: --cfg tokio_unstable -Dwarnings --cfg tokio_no_ipv6 --cfg tokio_no_tuning_tests ${{ matrix.rustflags }}
+          RUSTFLAGS: --cfg tokio_unstable -Dwarnings --cfg tokio_no_tuning_tests ${{ matrix.rustflags }}
```

Comment:
I just realized these `RUST*` flags should be set before `Swatinem/rust-cache@v2`:

```
 - name: Set RUST env
   run: |
     echo RUST_TEST_THREADS=1 >> "$GITHUB_ENV"
     echo 'RUSTFLAGS=--cfg tokio_unstable -Dwarnings --cfg tokio_no_tuning_tests ${{ matrix.rustflags }}' >> "$GITHUB_ENV"
```

Reply:
Considering how Swatinem/rust-cache works, it may be best for some workflows that meet certain conditions, but since we don't change rustflags often and sometimes set different rustflags for each step, I think it is also fine as-is.